data/_ui/interfaces.txt

```diff
@@ -118,7 +118,9 @@ color "overlay neutral shields" .43 .55 .70 .75
 color "overlay neutral hull" .45 .5 0. .25
 color "overlay neutral disabled" .5 0. 0. .25
 color "overlay outfit scan" .5 .5 .5 .25
+color "overlay outfit scan out of range" .25 .25 .25 .25
```

Comment:
I'd prefer `overlay outfit scan disabled`, just to be more concise.

Reply:
I don't think the difference in length between "out of range" and "disabled" (four characters total, two of which are spaces) is worth the loss of meaning.

Reply:
> I don't think the difference in length between "out of range" and "disabled" (four characters total, two of which are spaces) is worth the loss of meaning.

Is there a loss in meaning, though? If you're out of range scanning is disabled.

This color could also be used later by any other scan overlay things that also want a disabled color, there's no use in overly restricting ourselves.

Reply:
If "out of range" is more restrictive than "disabled" then there is by definition a difference in the meaning of the two, that is, by switching to "disabled" we lose the meaning that this is used when you are out of range.
Describing being out of range as being "disabled" is confusing to me, and inconsistent with other things. Weapons that are out of range are not disabled, for example.
If someone adds a concept of "disabled" scanners, then how that is handled should be a consideration for that time, not now.

Reply:
> If "out of range" is more restrictive than "disabled" then there is by definition a difference in the meaning of the two, that is, by switching to "disabled" we lose the meaning that this is used when you are out of range. Describing being out of range as being "disabled" is confusing to me, and inconsistent with other things. Weapons that are out of range are not disabled, for example. If someone adds a concept of "disabled" scanners, then how that is handled should be a consideration for that time, not now.

Well I mean you can't scan an out of range target, that seems disabled to me. Anyways, "disabled" is a common way to mark inactive colors in ui in general, and makes a lot more sense to a person perusing color names than out of range does, at least to me.

Oh well, I'm not going to die on this hill but six whole words for just a color name is a bit ridiculous.